projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/credentials/JobCredentialsService.java

```diff
@@ -22,6 +22,8 @@
 import java.util.Collections;
 import java.util.Optional;
 
+import static graphql.Assert.assertTrue;
```

Comment:
That looks worng. Why JobCredentialService depends on anything in graphql package ?